### DISCUSSION TITLE
2G+ Ausnahmen Baden-Württemberg

### DISCUSSION THREAD

**Discussion by dzubbs** [Digitaler-Impfnachweis/covpass-android#131] [Ideas]
Hallo liebe Entwickler*innen,

Seit Kurzem gibt es einige Ausnahmen zur 2G+ Regelung in BW ([Link](https://sozialministerium.baden-wuerttemberg.de/de/service/presse/pressemitteilung/pid/ausnahmen-von-der-testpflicht-bei-2g-plus/) ).  Momentan sieht man allerdings mit der Covpasscheck app nur ob das Zertifikat gültig ist, aber nicht das Datum der Impfung. In der normalen Covpass App word das Datum zwar noch in der Maske angezeigt, die zum Scannen bestimmt ist, jedoch ist dies nicht der Fall bei der Warn App. Hierzu müsste man in der Warnapp auf den QR code klicken, was ja sensible Daten zeigen würde. Aus diesem Grund habe ich zwei Vorschläge:
1. Das Anzeigen des Impfdatums und ob es sich um die Zweit/Boosterapp handelt beim Scann.
2. Die Möglichkeit zur Überprüfung, ob das aktuelle Zertifikat unter die Ausnahmereglung der 2G+ Regelung in BW fällt.

Ich hoffe ich konnte mein Anliegen verständlich skizzieren. Bei Rückfragen stehe ich gerne zur Verfügung.

Beste Grüße
Marcel Dzubba

**Comment by Ein-Tim** [comment]
Same as #128 & https://github.com/Digitaler-Impfnachweis/covpass-ios/issues/73

**Comment by alexcimander** [comment]
Hi  Marcel,

danke für dein Feedback und deinen Umsetzungsvorschlag! Du hast recht, die momentane Lage in Deutschland ist mit dem aktuellen Stand der CovPassCheck nicht gerade vereinbar. Wir arbeiten unter Hochdruck an entsprechenden Verbesserungen welche die 2G+ und 2G+B Konzepte, sowie altbekannte 3G und 2G Konzepte abdecken können. Dabei versuchen wir natürlich auch Möglichkeiten zur Überprüfung der "Ausnahmen" in Deutschland zu bieten.
Wir melden uns mit einem Update!

Falls Ihnen zukünftig noch weitere Ideen kommen oder du weitere Kritikpunkte hast, können Sie gerne unter "Issue" hier in Github mitdiskutieren. Dort werden Sie auf alle Fälle gesehen :-) 

Vielen Dank und Beste Grüße! Bleiben Sie gesund.
